jerry-port/common/jerry-port-context.c

```diff
@@ -27,13 +27,6 @@
  */
 static jerry_context_t *current_context_p = NULL;
 
-/**
```

Comment:
Why these are removed?

Reply:
Remove the comments in port impl, that's easily getting to in-consistence

Those already documented in header, document twice create a lot of inconsistence, and not updated. So just leave a single copy of the comment

Reply:
I agree with @lygstate, these comments should be place in the header file only.